src/data/faq.json

```diff
@@ -44,6 +44,17 @@
                                     "FAQ articles on general questions can be found <a href='#ramp_down_general' target='_blank'>here</a>, FAQ articles on implications regarding the app functions <a href='#ramp_down_effects' target='_blank'>here</a> and FAQ articles on alternatives <a href='#ramp_down_alternatives' target='_blank'>here</a>."
                                 ]
                             },
+                            {
+                                "title": "What can I do now? What do I need to know?",
+                                "anchor": "ramp_down_info",
+                                "active": false,
+                                "textblock": [
+                                    "After April 30, 2023, the Corona-Warn-App is going to trigger the technical instruction to stop the ENF (Exposure Notification Framework). This command will cause the smartphone’s operating system to stop exchanging random IDs and to delete all encounter information. ",

```

Comment:
Suggestion simplification:

"After April 30, 2023, the Corona-Warn-App is going to trigger the technical instruction to stop the ENF (Exposure Notification Framework)." =>

"After April 30, 2023, the Corona-Warn-App will deactivate the ENF (Exposure Notification Framework)." =>

Compare to German:

"Nach dem 30. April 2023 stehen die Funktionalitäten des Erkennens von Begegnungen mit anderen aktiven Corona-Warn-App-Systemen und die Erzeugung sowie der Austausch von Zufalls-IDs nicht mehr zur Verfügung. Diese Funktionalitäten werden deaktiviert."